Cargo.toml

```diff
@@ -1,4 +1,12 @@
+[workspace.package]
+authors = ["Parity Technologies <admin@parity.io>"]
```

Comment:
```suggestion
authors = ["Polkadot Fellowship"]
```

Reply:
When i change this now then the script will see a diff mismatch 🙈  
I will do it together with all the over followup fixes in the next MR.

Reply:
Yeah, I realized this as well :P I can also take this in my pr that adds a changelog.